incuna-sass/mixins/_svg-background.sass

```diff
@@ -1,12 +0,0 @@
-@mixin svg-background($svg, $inline: false, $width: '', $height: '', $dimensions: false)
```

Comment:
are we replacing this with the icon stuff? I am not sure if every svg should always be an icon or not? hmmm